main.py

```diff
@@ -170,6 +178,7 @@ def exit(self, *args):
     # Parse arguments
     parser = argparse.ArgumentParser(description='Helps stream desktop in Discord through virtual cams')
     parser.add_argument('--fps', dest='fps_rate', default="60", help="Sets FPS rate (default is 60)")
+    parser.add_argument('--flip', dest='flip', default="none", help="Flips the screen (options are 'h' and 'v') (default is none)")
```

Comment:
No need to use something like "none", an empty string is enough.
Change the description to something like
```
help="Flips the screen [h|v] (not flipped by default)"
```